component/Products/Product.html

```diff
@@ -0,0 +1,266 @@
+            </div>
+        </nav>
+
+        <section class="container flex">
+            <div class="sidebar">
+                <div class="filter_and_Clear flex">
+                    <b class="filter_type">Filter</b>
+                    <a href="#">clear</a>
+                </div>
+                <div class="product_price">
+                    <b class="filter_type">Price</b>
+                    <div class="ranges flex">
+                        <div>0</div>
+                        <div>500</div>
+                        <div>1000</div>
+                    </div>
+                    <div>
+                        <input class="range_input" type="range" id="vol" name="vol" min="0" max="50">
+                    </div>
+                </div>
+
+                <div class="product_category flex">
+                    <b class="filter_type">Category</b>
+                    <div>
+                        <input checked type="checkbox" id="checkbox1" name="checkbox1" value="checked">
+                        <label for="vehicle1">Men Clothing</label>
+                    </div>
+                    <div>
+                        <input checked type="checkbox" id="checkbox1" name="checkbox1">
+                        <label for="vehicle1">Men Clothing</label>
+                    </div>
+                </div>
+
+                <div class="product_rating flex">
+                    <b class="filter_type">Rating</b>
+                    <div>
+                        <input type="radio" id="html" name="fav_language" value="HTML">
+                        <label for="html">4 stars & above</label>
+                    </div>
+
+                    <div>
+                        <input type="radio" id="css" name="fav_language" value="CSS">
+                        <label for="css">3 stars & above</label>
+                    </div>
+
+                    <div>
+                        <input type="radio" id="javascript" name="fav_language" value="JavaScript">
+                        <label for="javascript">2 stars & above</label>
+                    </div>
+
+                    <div>
+                        <input type="radio" id="javascript" name="fav_language" value="JavaScript">
+                        <label for="javascript">1 star & above</label>
+                    </div>
+                </div>
+                
+                <div class="product_rating flex">
+                    <b class="filter_type">Sort by</b>
+                    <div>
+                        <input type="radio" id="html" name="sort" value="HTML">
+                        <label for="html">Price- Low to High</label>
+                    </div>
+
+                    <div>
+                        <input type="radio" id="css" name="sort" value="CSS">
+                        <label for="css">Price- High to Low</label>
+                    </div>
+                </div>
+            </div>
+            <div class="content flex wrap">
```

Comment:
You can use the grid for product display. It will be easier to manage cards that way and it will become responsive also.